### ISSUE TITLE
Suggestion to disable functionality temporarily while in Orbit or on the company moon

### ISSUE THREAD

**Issue by AshyDisaster** [NilaierMusic/NavigatingStars#3]
I keep getting bombarded by Path Not Found whenever I'm typing in terminal because I have to press the key the keybind is set to. The only feasible fix on my end is to just rebind it to a key that nothing uses like the arrow keys. Is this fixable or just mod incompatibility?

**Comment by NilaierMusic** [comment]
It's not hard to implement, just check when the player is in a terminal and disable the hotkey. It works, but i will push that with a bigger update, since i'm working on smoother animation, more lines and actual object avoidance, so the line wouldn't clip into stuff like the ground and stairs and then you have to wonder where it went. For now, you can probably just rebind the key, which isn't so convenient, but i guess works as a temporary workaround.

**Comment by AshyDisaster** [comment]
yeah rebinding it so far has been the best workaround; hasn't been that inconvenient. fine with waiting for a major update; thanks dev
